source/jupyter.rst

```diff
@@ -0,0 +1,148 @@
+.. jupyter:
+
+Installing Jupyter
+------------------
+
+Please see project Jupyter's `instructions for installation
+<http://jupyter.readthedocs.io/en/latest/install.html>`__. Basically,
+
+-  **If you already have Python (2 or 3)**, follow the following
+   instructions for your particular OS from the above webpage. We will
+   use Python 3 henceforth.
+
+-  **If you are a new Python user, or just want to reinstall Python
+   (say, to move from Python 2 to 3)** you can install Python+Jupyter
+   using `Anaconda <https://www.continuum.io/downloads>`__, by
+   downloading Anaconda's latest Python 3 version.
+
+Ubuntu / Linux
+~~~~~~~~~~~~~~
+
+As an existing Python 3 user, you may wish to install Jupyter using
+Python's package manager, ``pip``, instead of Anaconda. Do the
+following:
+
+.. code:: console
+
+   sudo pip3 install --upgrade pip
+
+   pip3 install jupyter
+
+If pip3 is not available, then first:
+
+.. code:: console
+
+   sudo apt-get install python3-pip
+
+If you keep getting a ``pip2 command not found`` error:
+
+.. code:: console
+
+   sudo apt remove python3-pip && sudo apt install python3-pip
+
+..
+
+   If you already have python 2.x and then installed python3, your pip
+   will be pointing to pip3. you can verify that by typing
+   ``pip --version`` which would be the same as ``pip3 --version``.
+
+Python 2.7 on Ubuntu 16.04
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If this does not work for python 2.7 on Ubuntu 16.04, do this:
+
+.. code:: console
+
+   sudo apt-get update
+
+   sudo apt-get -y install python2.7 python-pip python-dev
+
+   python --version
+
+   pip --version
+
+   sudo apt-get -y install ipython ipython-notebook
+
+   sudo -H pip install jupyter
+
+You will likely get an error like: "You are using pip version 8.1.1,
+however version 8.1.2 is available."
+
+.. code:: console
+
+
+   sudo -H pip install --upgrade pip
+
+   sudo -H pip install jupyter
+
+Mac
+^^^
+
+.. code:: console
+
+   pip3 install --upgrade pip
+
+   pip3 install jupyter
+
+Windows
+^^^^^^^
+
+Best option is to download and install Anaconda:
+https://www.continuum.io/downloads
```

Comment:
Please don't provide a second Python download instruction set (especially since this is not as complete as the Python instructions we already provide). Please instead link back to our Python install instructions.

Reply:
:+1: